Formula/o/oggz.rb

```diff
@@ -32,10 +32,10 @@ class Oggz < Formula
   depends_on "pkgconf" => :build
   depends_on "libogg"
 
-  # Fix -flat_namespace being used on Big Sur and later.
+  # build patch to include `<inttypes.h>` to fix missing printf format macros
   patch do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/03cf8088210822aa2c1ab544ed58ea04c897d9c4/libtool/configure-pre-0.4.2.418-big_sur.diff"
-    sha256 "83af02f2aa2b746bb7225872cab29a253264be49db0ecebb12f841562d9a2923"
+    url "https://raw.githubusercontent.com/Homebrew/formula-patches/c7dd59ab42edc3652529563bfb12ca9d1140c4af/liboggz/1.1.2-inttypes.patch"
+    sha256 "0ec758ab05982dc302592f3b328a7b7c47e60672ef7da1133bcbebc4413a20a3"
```

Comment:
Can we upstream this?

Reply:
yeah, I emailed the patch to `ogg-dev@xiph.org`